src/menu_nanox.c

```diff
@@ -0,0 +1,52 @@
+#include <os_io_seproxyhal.h> // todo put it somehow inside the guarded section
```

Comment:
todo is still valid?

Reply:
yes, I originally wanted to have the `#if defined(TARGET_NANOX)` on the very top of the file, but if I did that, it refused to compile with the message:
```
[LINK] bin/app.elf
obj/attestUtxo.o: In function `attestUtxo_handleInitAPDU':
/home/osboxes/git_projects/ledger/ledger-app-cardano/src/attestUtxo.c:369: undefined reference to `ui_displayBusy'
obj/signTx.o: In function `signTx_handleInput_ui_runStep':
/home/osboxes/git_projects/ledger/ledger-app-cardano/src/signTx.c:231: undefined reference to `ui_displayBusy'
obj/signTx.o: In function `signTx_handleInit_ui_runStep':
/home/osboxes/git_projects/ledger/ledger-app-cardano/src/signTx.c:133: undefined reference to `ui_displayBusy'
obj/signTx.o: In function `signTx_handleOutput_ui_runStep':
/home/osboxes/git_projects/ledger/ledger-app-cardano/src/signTx.c:376: undefined reference to `ui_displayBusy'
obj/signTx.o: In function `signTx_handleConfirm_ui_runStep':
/home/osboxes/git_projects/ledger/ledger-app-cardano/src/signTx.c:451: undefined reference to `ui_displayBusy'
obj/signTx.o:/home/osboxes/git_projects/ledger/ledger-app-cardano/src/signTx.c:534: more undefined references to `ui_displayBusy' follow
obj/uiHelpers.o: In function `ui_displayConfirm':
/home/osboxes/git_projects/ledger/ledger-app-cardano/src/uiHelpers.c:193: undefined reference to `ui_displayConfirm_init'
obj/uiHelpers.o: In function `ui_displayScrollingText':
/home/osboxes/git_projects/ledger/ledger-app-cardano/src/uiHelpers.c:237: undefined reference to `ui_displayScrollingText_init'
collect2: error: ld returned 1 exit status
/home/osboxes/bolos-devenv/nano_s/nanos-secure-sdk/Makefile.rules_generic:59: recipe for target 'bin/app.elf' failed
make: *** [bin/app.elf] Error 1
```
Seems like the linker skips the file if the seproxyhal import is not on top which bothered me